fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java

```diff
@@ -77,17 +77,19 @@ public class BrokerLoadJob extends BulkLoadJob {
     private Profile jobProfile;
     // If set to true, the profile of load job with be pushed to ProfileManager
     private boolean enableProfile = false;
+    private String cluster;
 
     // for log replay and unit test
     public BrokerLoadJob() {
         super(EtlJobType.BROKER);
     }
 
     public BrokerLoadJob(long dbId, String label, BrokerDesc brokerDesc,
-                         OriginStatement originStmt, UserIdentity userInfo)
+                         String cluster, OriginStatement originStmt, UserIdentity userInfo)
```

Comment:
Maybe we could merge the field `cluster` into BrokerDesc temporarily, and in our Load refactor task, field `cluster` can be included under the semantic of `resource_desc`, what's ur opinion?

If agreement reached, then the access to cluster can become getBrokerDesc().getCluster() (or getResourceDesc().getCluster() after our load refactor task).